src/AddTodoForm.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+
+    function handleAddTodo(event) {
+        event.preventDefault();
+        let todoTitle = event.target.elements.title.value;
+        props.onAddTodo(todoTitle);
+        console.log(todoTitle);
+        event.target.elements.title.value = '';
```

Comment:
You can also use event.target.title.value in line 7.
You can get rid of the console.log statement as well.
You could clear all your form inputs with the reset() method.